python/paddle/v2/fluid/framework.py

```diff
@@ -474,7 +473,7 @@ def find_name(var_list, name):
                 raise TypeError("'attrs' should be a dict.")
             for attr in proto.attrs:
                 attr_name = attr.name
-                if (not attr_name in attrs) or (attrs[attr_name] is None):
+                if (attr_name not in attrs) or (attrs[attr_name] is None):
```

Comment:
Hard to believe Python can do the same thing with so many different ways...

Reply:
From [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)
```
Use is not operator rather than not ... is. While both expressions are 
functionally identical, the former is more readable and preferred.

Yes:

if foo is not None:

No:

if not foo is None:
```

Reply:
It's better only one is allowed, but that's is not the case.